examples/google/README.adoc

```diff
@@ -0,0 +1,27 @@
+= Welcome to the Apache Camel Examples
+
+== Introduction
+
+This directory contains examples for Camel Google cloud components.
+View the individual example READMEs for details.
+
+== Examples
+
+[width="100%",cols="4,2,4",options="header"]
+|===
+| Example | Category | Description
+
+
+| link:google-pubsub/README.adoc[Google] (google) | Cloud | Multiple Camel Google examples
```

Comment:
The description should not be generic, it should be focused on Google PubSub

Reply:
You are right - ATM the google only contains the pubsub, can be changed once there are more google cloud services related component examples.

Reply:
changed

Reply:
Moreover the text of the link should mention PubSub too

Reply:
Sorry for that - my bad. Hope it should be OK now